docs/reference/resp2-to-resp3-replies.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "RESP2 to RESP3 replies migration guide"
+linkTitle: "RESP2 to RESP3 migration"
+weight: 4
+description: RESP2 to RESP3 replies reference for clients developers
+aliases:
+    - /topics/resp2-to-resp3-replies/
+---
+
+In the journey from RESP2 to [RESP3](https://redis.io/docs/reference/protocol-spec/), one of the primary motivations was 
+to streamline the developer experience by simplifying response parsing. Over time, it became evident that developers 
+frequently performed specific data transformations on RESP2 replies. Addressing these transformations directly at the 
+protocol level not only reduces the overhead for developers but also ensures a more consistent and efficient data 
+handling process. This documentation provides a reference for developers to migrate their clients from RESP2 to RESP3.
+
+### Command Replies Comparison
+- If you see any missing commands, please [open an issue](https://github.com/redis/redis-doc/issues/new?title=RESP2-to-RESP3%20command%20is%20missing)
+- `null` was "promoted" from a subtype of `BlobString` and `Array` to its own type, these changes are not included in this list.
```

Comment:
is this suppose to represent a general list of changes between RESP2 and RESP2?
in which case many lines are missing (double, boolean, map, etc).
i don't think we should repeat it, but rather add a prompt line that redirects to the spec.

instead, since the tables below only describe commands with non-trivial changes, this block could be a prompt statement about generic transformation, like an interlaced array becoming a map, and clearly stating that the list below covers only the non-trivial changes.

alternatively, if we want, we can put here a full list of the common transformation (like interlaced array to map, string to double, int to boolean), i.e. anything that can generically be applied, so that we don't need to list all the affected commands.

Reply:
Good point. Updated